bumble/rfcomm.py

```diff
@@ -538,8 +538,9 @@ def on_uih_frame(self, frame: RFCOMM_Frame) -> None:
             f'[{self.dlci}] {len(data)} bytes, '
             f'rx_credits={self.rx_credits}: {data.hex()}'
         )
-        if len(data) and self.sink:
-            self.sink(data)  # pylint: disable=not-callable
+        if data:
+            if self.sink:
+                self.sink(data)  # pylint: disable=not-callable
```

Comment:
Since we have annotation now, it is no longer needed.
```suggestion
                self.sink(data)
```

Reply:
I still get a pylint error without that comment:
`bumble/rfcomm.py:543:16: E1102: self.sink is not callable (not-callable)`